docs/contribute/contribution.data.js

```diff
@@ -0,0 +1,13 @@
+import { createMarkdownRenderer } from 'vitepress'
+
+export default {
+
```

Comment:
I pushed a solution based on the following links and suggest approaches I found for the english version so you have an example:

- https://vitepress.dev/guide/data-loading
- https://github.com/vuejs/vitepress/discussions/3049
- https://github.com/vuejs/vitepress/discussions/2921#discussioncomment-7023589 

But I am not really convinced and as of now it seems to me a bit of an overkill to introduce a data loader for this one markdown file and an extra section. 

Wdyt @alvarosabu ? 

I suggest we stick with the other approach that Vitepress is already using. :)